sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/UcfgGenerator.cs

```diff
@@ -95,13 +95,16 @@ protected override void Initialize(SonarAnalysisContext context)
 
                     cc.RegisterSyntaxNodeActionInNonGenerated(
                         c => WriteUCFG<BaseMethodDeclarationSyntax>(c, x => x.Body),
-                        SyntaxKind.ConstructorDeclaration,
-                        SyntaxKind.OperatorDeclaration);
+                        SyntaxKind.ConstructorDeclaration);
```

Comment:
`WriteUCFG<BaseMethodDeclarationSyntax>` should be better with `WriteUCFG<ConstructorDeclarationSyntax>`